src/Sentry.AspNetCore/SentryAspNetCoreOptions.cs

```diff
@@ -43,6 +43,12 @@ public class SentryAspNetCoreOptions : SentryLoggingOptions
     /// </summary>
     public TimeSpan FlushTimeout { get; set; } = TimeSpan.FromSeconds(2);
 
+    /// <summary>
+    /// RouteName sets the transaction name that was created by the ASPNetCore integration
+    /// when the integration isn't able to define the transaction name..
+    /// </summary>
+    public ISentryRouteName? RouteName { get; set; }
```

Comment:
Something that better reflects what this options does:

```suggestion
    public ITransactionNameProvider? TransactionNameProvider { get; set; }
```